tasmota/berry/drivers/Core200S.be

```diff
@@ -0,0 +1,712 @@
+import string
+import webserver
+
+class StateEnum
+    var value_map
+    var value
+
+    def init(value_map, value)
+        self.value_map = value_map
+        if type(value) == 'int'
+            for k : self.value_map.keys()
```

Comment:
`for v: value_map` allows to iterate already on map values. So you can simplify code